cafebabel/static/js/script.js

```diff
@@ -113,3 +113,38 @@ Array.from(document.querySelectorAll('h1.edit input')).forEach(inputh1 => {
     () => inputh1.value || h1.classList.remove('active')
   )
 })
+
+/* Auto-expand textarea when typing http://jsfiddle.net/hmelenok/WM6Gq/ */
+var textAreaAutoResize
+if (window.attachEvent) {
```

Comment:
Is that test consistent with other scripts?

Reply:
It works, that's all. I guess it could be cleaner. It that what you mean ?

Reply:
Right, I think we only use `addEventListener` and we don't make backward compatibility to IE5/8

Reply:
Ah sorry I got what you mean. It could be in a separate script. A script only called on article edit pages

Reply:
No, I mean we did not take care of that retro-compatibility since now.

Reply:
Ok you'll explain that later

Reply:
@cafebabel-webmaster https://stackoverflow.com/a/30915025/328117.
We don't treat older browsers cases.

Reply:
ok